spring-integration-zip/src/main/java/org/springframework/integration/zip/splitter/UnZipResultSplitter.java

```diff
@@ -16,15 +16,16 @@
 
 package org.springframework.integration.zip.splitter;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
-
 import org.apache.commons.io.FilenameUtils;
 import org.springframework.integration.file.FileHeaders;
 import org.springframework.integration.support.MessageBuilder;
 import org.springframework.integration.zip.ZipHeaders;
 import org.springframework.messaging.Message;
+import org.springframework.messaging.MessageHeaders;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
```

Comment:
Hello @englishman !

Thank you for the contribution!

I will review this in nit-pick manner with the hope to avoid similar comment in the future.
Just for case to save your and our time!